{
  "artifacts": [],
  "command": "my_main",
  "experiment": {
    "base_dir": "/workspace/pymarl/src",
    "dependencies": [
      "numpy==1.24.4",
      "PyYAML==5.4.1",
      "sacred==0.8.5",
      "torch==2.1.0"
    ],
    "mainfile": "main.py",
    "name": "pymarl",
    "repositories": [],
    "sources": [
      [
        "main.py",
        "_sources/main_63d7d00072f4e5be0310acb11aa60581.py"
      ],
      [
        "run.py",
        "_sources/run_7911ddab0b7ff30dd1480a75183ffc39.py"
      ],
      [
        "utils/logging.py",
        "_sources/logging_aa14e56e60b01dd7a9ddb8e6cdf36b85.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/usr/local/lib/python3.8/dist-packages/sacred/config/captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"src/main.py\", line 35, in my_main\n    run(_run, config, _log)\n",
    "  File \"/workspace/pymarl/src/run.py\", line 48, in run\n    run_sequential(args=args, logger=logger)\n",
    "  File \"/workspace/pymarl/src/run.py\", line 166, in run_sequential\n    episode_batch = runner.run(test_mode=False)\n",
    "  File \"/workspace/pymarl/src/runners/episode_runner.py\", line 49, in run\n    self.reset()\n",
    "  File \"/workspace/pymarl/src/runners/episode_runner.py\", line 45, in reset\n    self.env.reset()\n",
    "  File \"/usr/local/lib/python3.8/dist-packages/smac/env/starcraft2/starcraft2.py\", line 403, in reset\n    self._launch()\n",
    "  File \"/usr/local/lib/python3.8/dist-packages/smac/env/starcraft2/starcraft2.py\", line 318, in _launch\n    self._run_config = run_configs.get(version=self.game_version)\n",
    "  File \"/usr/local/lib/python3.8/dist-packages/pysc2/run_configs/__init__.py\", line 36, in get\n    return max(configs.values(), key=lambda c: c.priority())(version=version)\n",
    "  File \"/usr/local/lib/python3.8/dist-packages/pysc2/run_configs/platforms.py\", line 175, in __init__\n    super(Linux, self).__init__(base_dir, \"SC2_x64\", version=version, env=env)\n",
    "  File \"/usr/local/lib/python3.8/dist-packages/pysc2/run_configs/platforms.py\", line 57, in __init__\n    super(LocalBase, self).__init__(\n",
    "  File \"/usr/local/lib/python3.8/dist-packages/pysc2/run_configs/lib.py\", line 128, in __init__\n    self.version = self._get_version(version)\n",
    "  File \"/usr/local/lib/python3.8/dist-packages/pysc2/run_configs/lib.py\", line 236, in _get_version\n    versions = self.get_versions(containing=game_version)\n",
    "  File \"/usr/local/lib/python3.8/dist-packages/pysc2/run_configs/platforms.py\", line 90, in get_versions\n    for v in os.listdir(versions_dir)\n",
    "FileNotFoundError: [Errno 2] No such file or directory: '/pymarl/3rdparty/StarCraftII/Versions'\n"
  ],
  "heartbeat": "2024-05-03T10:27:38.395066",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Core(TM) i5-10400F CPU @ 2.90GHz",
    "hostname": "7eccbc0c7b12",
    "os": [
      "Linux",
      "Linux-5.15.146.1-microsoft-standard-WSL2-x86_64-with-glibc2.29"
    ],
    "python_version": "3.8.10"
  },
  "meta": {
    "command": "my_main",
    "config_updates": {
      "env_args": {
        "map_name": "2s3z"
      }
    },
    "named_configs": [],
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--id": null,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [
        "env_args.map_name=2s3z"
      ],
      "help": false,
      "with": true
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2024-05-03T10:27:37.744982",
  "status": "FAILED",
  "stop_time": "2024-05-03T10:27:38.407867"
}